server/views/pages/view/cases.njk

```diff
@@ -39,7 +39,11 @@
     {% if case.probationPractitioner %}
         {% set comDetailsHtml = '<a class="govuk-link" href="/licence/view/probation-practitioner/staffCode/' + case.probationPractitioner.staffCode + '">' + case.probationPractitioner.name + '</a>' %}
     {% else %}
-        {% set comDetailsHtml = 'Unallocated' %}
+        {% if case.kind == "TIME_SERVED" or case.hardStopKind == "TIME_SERVED"%}
```

Comment:
Q - I take it both are needed as this was the reason it wasn't working on Friday